peps/pep-0772.rst

```diff
@@ -168,11 +168,10 @@ publicly in a timely fashion.
 Delegations
 -----------
 
-The Python Steering Council is expected to delegate decision making to the
-Packaging Council for PEPs related to the Python packaging.  The bodies would
-work together on issues that intersect the packaging domain and language
-stewardship (including the CPython implementation, standard library, and
-distribution).
+The Python Steering Council will delegate decision making to the Packaging
+Council for PEPs related to the Python packaging. The bodies would work
+together on issues that intersect the packaging domain and language stewardship
+(including the CPython implementation, standard library, and distribution).
 
 The PSF Board is encouraged to formally deprecate the Packaging Workgroup and
```

Comment:
"close" or "end" instead of "deprecate"?

Reply:
Or "deactivate"? Following the language of https://www.python.org/psf/workgroups/

Reply:
Agreed that "deprecate" doesn't have the finality and closure I think we expect.  @hugovk I'd be fine with "deactivate" although I can't find that word used at the workgroups link.